packages/block-library/src/avatar/block.json

```
@@ -29,8 +29,8 @@
 		"align": true,
 		"alignWide": false,
 		"spacing": {
-			"margin": true,
-			"padding": true
+			"margin": false,
+			"padding": false
```

What @richtabor suggested in the issue is this here:

```suggestion
			"margin": true,
			"padding": true,
			"__experimentalDefaultControls": {
				"margin": false,
				"padding": false
			}
```